itests/org.openhab.core.voice.tests/src/main/java/org/openhab/core/voice/voiceconsolecommandextension/VoiceConsoleCommandExtensionTest.java

```diff
@@ -16,10 +16,10 @@
 
 import org.junit.jupiter.api.BeforeEach;
 import org.openhab.core.audio.AudioManager;
+import org.openhab.core.audio.internal.AudioManagerImpl;
```

Comment:
It seems the resolution issues can be resolved by removing this internal import. After that the tests run again but still fail.